IHP/IDE/SchemaDesigner/Types.hs

```diff
@@ -8,9 +8,9 @@ module IHP.IDE.SchemaDesigner.Types where
 import IHP.Prelude
 
 data Statement
-    = 
+    =
     -- | CREATE TABLE name ( columns );
-    CreateTable { name :: Text, columns :: [Column], constraints :: [Constraint] }
+      StatementCreateTable { unsafeGetCreateTable :: CreateTable }
```

Comment:
What is the reason behind this refactoring? :) IMO this new level of indirection makes the code a bit more complex